nasa/harmony
# DAS-1714 - Update NetCDF-to-Zarr Docker image reference to point to ghcr.io.

## Jira Issue ID

DAS-1714

## Description

This PR updates the default Harmony environment variables to direct `HARMONY_NETCDF_TO_ZARR_IMAGE` to [the new image hosted in ghcr.io](https://github.com/nasa/harmony-netcdf-to-zarr/pkgs/container/harmony-netcdf-to-zarr).

There will be an accompanying change to the `harmony-ci-cd` repository, and I'll add environment variables to the Bamboo deployment project, too.

## Local Test Steps

* Pull this branch.
* Ensure `docker pull` will work on the new path in `env-defaults`.
* Ensure a local Harmony instance can be created via `bin/bootstrap-harmony`.
* The following request should run and succeed:

```
localhost:3000/C1245618475-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&maxResults=1&concatenate=true&format=application%2Fx-zarr 
```

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
